pyhf/utils.py

```diff
@@ -127,27 +127,34 @@ def pvals_from_teststat(sqrtqmu_v, sqrtqmuA_v):
     return CLsb, CLb, CLs
 
 
-def runOnePoint(muTest, data, pdf, init_pars=None, par_bounds=None):
+def cls(mu_test, data, pdf, init_pars=None, par_bounds=None, **kwargs):
     r"""
-    Computes test statistics (and expected statistics) for a single value
-    of the parameter of interest
+    Computes test statistics (and expected statistics) for a single value of the parameter of interest
 
     Args:
-        muTest (Number or Tensor): The value of the parameter of interest (POI)
+        mu_test (Number or Tensor): The value of the parameter of interest (POI)
         data (Number or Tensor): The root of the calculated test statistic given the Asimov data, :math:`\sqrt{q_{\mu,A}}`
-        init_pars (Array or Tensor): the initial parameter values to be used for minimization
-        par_bounds (Array or Tensor): the parameter value bounds to be used for minimization
+        init_pars (Array or Tensor): The initial parameter values to be used for minimization
+        par_bounds (Array or Tensor): The parameter value bounds to be used for minimization
+
+    Keyword Args:
+        observed_set (bool): Bool for returning :math:`CL_{s+b}` and :math:`CL_{b}`
+        expected (bool): Bool for returning :math:`CL_{\textrm{exp}}`
+        expected_set (bool): Bool for returning the :math:`(-2,-1,0,1,2)\sigma` :math:`CL_{\textrm{exp}}` --- the "Brazil band"
+        test_statistics (bool): Bool for returning :math:`q_{\mu}` and :math:`q_{\mu,A}`
```

Comment:
should these be called `return_X` ? similar to sklearn?

Reply:
Yeah, I had originally considered that as unnecessarily long, but The Zen of Python does tell us that

> Explicit is better than implicit.

so might as well follow sklearn's standard.

Reply:
Changed in https://github.com/diana-hep/pyhf/pull/341/commits/c511057d206a6837d6a6bf5e00e64e2b28f4fa1a

Reply:
thanks!